lighthouse-cli/test/smokehouse/run-smoke.js

```diff
@@ -88,18 +89,15 @@ const SMOKETESTS = [{
 
 /**
  * Display smokehouse output from child process
- * @param {{id: string, process: NodeJS.Process} || {id: string, error: Error & {stdout : NodeJS.WriteStream, stderr: NodeJS.WriteStream}}} result
+ * @param {{id: string, stdout : string, stderr: string, error?: Error}} result
  */
 function displaySmokehouseOutput(result) {
   console.log(`\n${purpleify(result.id)} smoketest results:`);
   if (result.error) {
     console.log(result.error.message);
-    process.stdout.write(result.error.stdout);
-    process.stderr.write(result.error.stderr);
-  } else {
-    process.stdout.write(result.process.stdout);
-    process.stderr.write(result.process.stderr);
   }
+  process.stdout.write(result.stdout);
+  process.stderr.write(result.stderr);
```

Comment:
Lol nice.

Reply:
sometimes you just gotta go to big G unfortch 🧹

Reply:
![image](https://user-images.githubusercontent.com/4071474/50787963-d45ad000-126c-11e9-9a3b-09f358aaef0f.png)

i'm gonna need some help here Patrick =P

Reply:
lol sorry I was replying to 

> is there not a broom/mop emoji on gh? I'm sad about this.

big G being Google :) i.e. I got the broom emoji by googling "broom emoji" and copy/pasting

Reply:
oooooh ok. I just see a square :) 
![image](https://user-images.githubusercontent.com/4071474/50789383-eb032600-1270-11e9-870b-e7b981a97244.png)

Reply:
![image](https://user-images.githubusercontent.com/4071474/50789410-feae8c80-1270-11e9-9464-f7a45c025fa6.png)

weird!